.travis.yml

```diff
@@ -7,7 +7,10 @@ python:
 
 os: linux
 
-dist: trusty
+dist: xenial
```

Comment:
why do you want to switch from 14.04? Do the changes not allow us to support it anymore?

Reply:
`xvfb` service is only available on 16.04 and above so I had to bump the version up. Meanwhile, 14.04 came to its end of life cycle this April so it is probably a good idea for us to move on to something newer, assuming this does not break anything in the process.